modules/psap-special-resource-operator-using.adoc

```diff
@@ -0,0 +1,378 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="using-the-special-resource-operator"]
+= Using the Special Resource Operator
+
+As an example, the Special Resource Operator can be used to manage the build and deployment of a driver container for a minimal kernel module called simple-kmod. 
+
+The objects required to build and deploy the kernel module can be defined using a Helm chart. The SRO image contains a local repository of Helm charts including the templates for deploying the simple-kmod. SRO can also read the templates from a ConfigMap. Both methods are demonstrated in the following examples.
+
+[id="deploy-simple-kmod-using-local-chart"]
+== Build and run the simple-kmod SpecialResource using the SRO image's local manifests.
+
+.Prerequisites
+
+* You have a running {product-title} cluster
+* You have installed the OpenShift CLI (`oc`).
+* You are logged into the OpenShift CLI as a user with `cluster-admin` privileges.
+* You have installed the xref:../psap-node-feature-discovery#installing-the-node-feature-discovery-operator_node-feature-discovery-operator[Node Feature Discovery (NFD) Operator]
+* You have installed the Special Resource Operator.
+
+.Procedure
+. To deploy the simple-kmod using the SRO image's local Helm repository, the following SpecialResource manifest can be used. Save this YAML as `simple-kmod-local.yaml`
++
+[source,yaml]
+----
+apiVersion: sro.openshift.io/v1beta1
+kind: SpecialResource
+metadata:
+  name: simple-kmod
+spec:
+  namespace: simple-kmod
+  chart:
+    name: simple-kmod
+    version: 0.0.1
+    repository:
+      name: example
+      url: file:///charts/example
+  set:
+    kind: Values
+    apiVersion: sro.openshift.io/v1beta1
+    kmodNames: ["simple-kmod", "simple-procfs-kmod"]
+    buildArgs:
+    - name: "KMODVER"
+      value: "SRO"
+  driverContainer:
+    source:
+      git:
+        ref: "master"
+        uri: "https://github.com/openshift-psap/kvc-simple-kmod.git"
+----
+
+. Create the SpecialResource with:
++
+[source,terminal]
+----
+$ oc create -f simple-kmod-local.yaml
+----
+
+. The simple-kmod resources will be deployed in the namespace `simple-kmod` as specified in the object manifest. After a short time, the build Pod for the simple-kmod driver container should start running. After a few minutes, the build should complete and the driver container Pods should start Running.
```

Comment:
How about:
The simple-kmod resources are deployed in the `simple-kmod` namespace as specified in the object manifest. After a short time, the build pod for the simple-kmod driver container runs. The build completes after a few minutes, then the driver container pods run.

Reply:
Pod should be lowercase (pod) unless you are specifically interacting with the API Pod object.

Reply:
Fixed, thanks